proxy.js

```diff
@@ -803,7 +812,9 @@ function Miner(id, params, ip, pushMessage, portData, minerSocket) {
             lastShare: this.lastShareTime,
             coin: this.coin,
             pool: this.pool,
-            id: this.id
+            id: this.id,
+            user: this.user,
```

Comment:
Why this is needed to be stored? I think it is enough to check isAllowedLogin(this.user, this.password) in Miner function.